tests/testthat/test-robin_lm.R

```diff
@@ -0,0 +1,28 @@
+# h_interaction ----
+
+test_that("h_interaction works correctly", {
+  expect_false(h_interaction(y ~ trt + z, treatment = trt ~ x))
+  expect_true(h_interaction(y ~ trt:z, treatment = trt ~ x))
+  expect_true(h_interaction(trt * y ~ trt:z, treatment = trt ~ x))
+  expect_true(h_interaction(y ~ trt:z, treatment = trt ~ 1))
+})
+
```

Comment:
the test is already added in another file

Reply:
Agree and remove it from test-robin_glm.R